tools/ColorTool/ColorTool/Program.cs

```diff
@@ -304,7 +304,7 @@ private static IntPtr GetStdOutputHandle()
 
         static void PrintSchemes()
         {
-            var schemeDirectory = new FileInfo(new Uri(Assembly.GetEntryAssembly().GetName().CodeBase).AbsolutePath).Directory.FullName + "/schemes"; 
+            var schemeDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "schemes\\");
```

Comment:
is the trailing `\\` a cross platform issue? Is it better to remove it, or use `System.IO.Path.DirectorySeparatorChar` if it's actually needed?

Reply:
Actually, yeah -- this is a good question. Using `Path.Combine` followed by manually putting a `\\` on the end seems like an antipattern.